### PR TITLE
feat: add missing storage bucket

### LINKED ISSUE #780: feat: add missing security-incident-log-bucket to core-landing-zone exp solution
Add missing security-incident-log-bucket to the experimentation core-landing-zone solution.

### PR DESCRIPTION
**Pull request by lucstjean-ssc**
Closes #780 

Add missing security-incident-log-bucket to the experimentation core-landing-zone solution.

Permissions for bucket added via previous issue #775.

### CODE REVIEW

File: solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  namespace: logging
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${logging-project-id} # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${logging-project-id}
+    cnrm.cloud.google.com/project-id: logging-project-id # kpt-set: ${logging-project-id}
+spec:
+  # enable autoclass
+  # https://cloud.google.com/storage/docs/autoclass
+  autoclass:
+    enabled: true
+  location: northamerica-northeast1
```
Comment (tackaberry) [inline solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml:30]: It seems like this isn't consistent across the code base, however, should we not have `# kpt-set: ${region}` on the region?

Comment (lucstjean-ssc) [reply solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml:30]: Ideally, yes, I agree, but that was not the decision taken. We've pinned every resource to northamerica-northeast1.

**Review by fmichaelobrien — APPROVED** [review]
/lgtm
- hardcoding parameterization can be a 2nd jira
- Montreal is the default

File: solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml (hunk 1 above)
Comment (tackaberry) [reply solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/cloud-storage-buckets.yaml:30]: OK.   We can address later.

**Review by tackaberry — APPROVED** [review]
lgtm

**Review by alaincormier-ssc — APPROVED** [review]

**Comment by fmichaelobrien** [comment]
full region parameterization side issue here
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/791

**Pull request merged**